pkg/koordlet/metrics/cpu_burst.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+var (
+	ContainerCFSBurstCPU = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Subsystem: KoordletSubsystem,
+		Name:      "container_cfs_burst_cpu_cores",
```

Comment:
This metric records the value of container cgroup's `cpu.cfs_burst_us` but not the value of the container's bursty cores.
e.g. `cpu.cfs_burst_us=1000000, cpu.cfs_period_us=100000` means the container has a bursty slice as 1000000/100000=10 cores.

Reply:
Yes, you are right. BTW, you are observing very carefully.